api/models/enums.js

```diff
@@ -33,11 +32,11 @@ module.exports.RESOURCE_TYPES = [
   'Other',
 ];
 module.exports.RESOURCE_PATHS = [
-  'Designer Path',
-  'Developer Path',
-  'Policymaker Path',
-  'Risk Manager Path',
-  'Explorer Path',
+  'Designer',
+  'Developer',
+  'Policymaker',
+  'Risk Manager',
+  'Explorer',
   'Other',
 ];
```

Comment:
This is going to be a pain to update since I already filled prod. Let's keep these enums the same. If you want to remove "Path", let's just do it on the UI side.